components/AssetPage/Download/DownloadOptions.tsx

```diff
@@ -71,16 +71,20 @@ const DownloadOptions = ({ open, assetID, tempUUID, files, res, fmt, selectMap,
           !threeDFormats.includes(m) || fmt === 'zip' ?
             m !== 'nor_dx' || norMode === 'dx' || fmt === 'zip' ?
               m !== 'nor_gl' || norMode === 'gl' || fmt === 'zip' ?
-                <DownloadMap
-                  key={i}
-                  name={m}
-                  res={res}
-                  fmt={fmt}
-                  type={type}
-                  data={files[m][res]}
-                  trackDownload={trackDownload}
-                  selectMap={selectMap}
-                />
+                // TODO: should we make it possible to download the bundle
+                // without the preview in it?
+                m !== 'preview' ?
```

Comment:
I don't understand what this addition does - making the change here locally doesn't seem to do anything?

Reply:
I get this error if I remove this condition:
![image](https://user-images.githubusercontent.com/43729152/200914894-b9a5132c-ec92-4769-afae-a5f2576c08bb.png)

The value of the `data` attribute of `DownloadMap` in this case is `undefined`, because it's supposed to be `files[m][res]` ([see here](https://github.com/fulopkovacs/polyhaven.com/blob/include-preview-png-in-zip-download/components/AssetPage/Download/DownloadOptions.tsx#L81)), but `files["preview"]` has only these keys: `"url"`, `"path"`, `"size"`, no resolutions.